### PR TITLE
Cpu caps fixup

### PR DESCRIPTION
**Pull request by liujunming**

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: hypervisor/arch/x86/cpu_caps.c [hunk 1]
```diff
@@ -118,7 +107,7 @@ bool is_apl_platform(void)
 {
 	bool ret = false;
 
-	if ((boot_cpu_data.displayfamily == 0x6U) && (boot_cpu_data.displaymodel == 0x92U)) {
+	if ((boot_cpu_data.displayfamily == 0x6U) && (boot_cpu_data.displaymodel == 0x5c)) {
```
Comment (dongyaozu) [inline hypervisor/arch/x86/cpu_caps.c:110]: Any reason for this change? It is completely different.
BTW 0x5c -->0x5CU ?

Comment (liujunming) [reply hypervisor/arch/x86/cpu_caps.c:110]: Hi, Eddie.
From SDM,
`06_5CH Intel Atom processors based on Goldmont Microarchitecture`
APL platform's DisplayModel is 0x5c, `0x92U` was a bug.

I will change 0x5c -->0x5cU

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by liujunming** [comment]
[External_System_ID] ACRN-5175

**Pull request merged**